src/phrases/phrases-nb.ts

```diff
@@ -1144,6 +1144,9 @@ const phrases = {
     },
     grepCodes: 'Følgende koder har feil format eller eksisterer ikke, og vil ikke bli lagt til: ',
   },
+  warningMessage: {
+    fieldWithWrongLanguage: 'Dette feltet er hentet fra språkkode: {{language}}',
```

Comment:
```suggestion
    fieldWithWrongLanguage: 'Innholdet er hentet fra språkkode: {{language}}',
```

Reply:
Kanskje det er greiere med "Dette feltet"? Da er vi mer eksplisitte på at det bare er akkurat dette feltet som er hentet fra $LANG

Reply:
Det er greit for meg. Eg tenkte berre at teksten uansett står knytta til et felt slik at du skjønner det fra konteksten. Men skal ikkje være vanskelig.